savaslabs/savaslabs.github.io
# Issue #9616: Update team headshots

Fixes issue [#9616](https://pm.savaslabs.com/issues/9616)

## Summary of changes

1. Replaces various company and square-outside headshots for various team members. 

## Notes

New images for comparison can be found in the task.

## To test

Confirm the following have been updated on the [company page:](https://stage3--savas-staging.netlify.com/company/)
- [ ] Alex
- [ ] Devan
- [ ] Kayla
- [ ] Sean
- [ ] Maddy
- [ ] Madeline

Confirm the following have been updated on the individual team member pages:
- [ ] Alex's [background](https://stage3--savas-staging.netlify.com/company/alex-manzo/) headshot
- [ ] Devan's [background](https://stage3--savas-staging.netlify.com/company/devan-chase/) headshot
- [ ] Sean's [background](https://stage3--savas-staging.netlify.com/company/sean-oshea/) headshot
- [ ] Maddy's [background](https://stage3--savas-staging.netlify.com/company/maddy-bishop-van-horn/) headshot
- [ ] Madeline's [background](https://stage3--savas-staging.netlify.com/company/madeline-streilein/) headshot

### Pull request reviewer

As the reviewer, I have verified the following:
